pkg/server/check.go

```diff
@@ -75,6 +74,7 @@ func (s *Server) Check(ctx context.Context, req *openfgav1.CheckRequest) (*openf
 		ContextualTuples: req.GetContextualTuples(),
 		Context:          req.GetContext(),
 		Consistency:      req.GetConsistency(),
+		Typesys:          typesys,
```

Comment:
Why are we making this change in this PR?

Reply:
Removed 👍